### PR TITLE
Fix: upgrade sentry integration

### PR DESCRIPTION
**Pull request by ThisIsMissEm**
## Description

The modules for the sentry integration were outdated, and it turns out there's now a specific nextjs configuration that exists, so we were able to move to that relatively easily.

## Testing

Have run unit tests, and will also test that the app functions as expected on vercel

## Commit checklist

- [x] All acceptance criteria are met.
- [x] Includes tests to ensure functionality and prevent regressions.
- [x] Relevant documentation, if any, has been written/updated.
- [x] The changelog has been updated, if applicable.

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -19,50 +19,28 @@
...
+  // Additional config options for the Sentry Webpack plugin. Keep in mind that
+  // the following options are set automatically, and overriding them is not
+  // recommended:
+  //   release, url, org, project, authToken, configFile, stripPrefix,
+  //   urlPrefix, include, ignore
 
-    return config;
-  },
+  // Suppresses all logs
+  silent: true,
```
Comment (solid-akb) [inline next.config.js:39]: Just confirming, this `silent:true` is only for the webpack logs, the nextjs ones  still go through?

File: package.json [hunk 2]
```diff
@@ -104,4 +102,4 @@
     "vercel": "^21.2.3",
     "whatwg-fetch": "^3.5.0"
   }
-}
\ No newline at end of file
+}
```
Comment (solid-akb) [inline package.json:105]: is this a prettier thing? Is it possible to remove whitespace?

**Review by solid-akb — APPROVED** [review]
Thanks for finding this and implementing! Left a few small comments/questions but otherwise approve!

File: next.config.js (hunk 1 above)
Comment (ThisIsMissEm) [reply next.config.js:39]: This was the configuration that was recommended by Sentry, also it only affects the sentry integration, as it's in `sentryWebpackPluginOptions`

File: package.json (hunk 2 above)
Comment (ThisIsMissEm) [reply package.json:105]: it is a prettier thing, but in general, all files should end with a trailing whitespace.

**Pull request merged**